src/styles/blocks/card.scss

```
@@ -0,0 +1,120 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background-color: $white-color;
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+
+  @include content-inline-padding;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 32px 0 40px;
```

it's not a top image margin, it should be a 32px of top card padding
```suggestion
    margin: 0 auto 40px;
```